lib/id3c/cli/command/etl/__init__.py

```diff
@@ -244,7 +244,8 @@ def find_sample(db: DatabaseSession, identifier: str, for_update = True) -> Any:
         """ + query_ending, (identifier,identifier,))
 
     if not sample:
-        LOG.info(f"No sample with identifier «{identifier}» found")
+        LOG.info(f"No sample found with identifier «{identifier}» or collection identifier «{collection_identifier}»")
```

Comment:
There is no local variable collection_identifier here.

Reply:
Fixing!